### DISCUSSION TITLE
exclude paths from prepend

### DISCUSSION THREAD

**Discussion by surendra-iouring** [iouring-engineering/openapi-merge#9] [Ideas]
An existing prepend feature is great option for adding some basic path in all the API's however in our case we may have to ignore some of paths from adding prepend paths, can we have any solution for this?

**Comment by hari-iouring** [comment]
@surendra-iouring  - please create this as a feature request

**Reply by surendra-iouring** [reply]
@hari-iouring thanks a lot
